presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenState.kt

```diff
@@ -6,5 +6,6 @@ import me.androidbox.component.agenda.VisitorType
 
 data class EventScreenState(
     val listOfPhotoUri: List<Uri> = mutableStateListOf<Uri>(),
-    val selectedVisitorType: VisitorType = VisitorType.ALL
+    val selectedVisitorType: VisitorType = VisitorType.ALL,
+    val selectedVisitor: String = "",
```

Comment:
Isn't there more to a visitor than just the name (assuming this string refers to the name which isn't clear)? If this refers to the name, I'd make it refer to the ID since names can appear multiple times. And if it refers to the ID, I'd also name it that way (`selectedVisitorId`)